### DISCUSSION TITLE
Codeigniter 4 Error saat Push ke Heroku - Topidesta - Tulisan Programmer Gadungan

### DISCUSSION THREAD

**Discussion by giscus** [topidesta/topidesta#2] [Comments]
# Codeigniter 4 Error saat Push ke Heroku - Topidesta - Tulisan Programmer Gadungan

Bismillah .... Kebetulan lagi handle Project yang sudah gak dihandle sama Maintenernya, tapi lagi gw tanya, mau dilanjutin apa kagak sih…

https://topidesta.my.id/codeigniter-4-error-saat-push-ke-heroku

**Comment by mdestafadilah** [comment]
test

**Reply by muhammad-rifqi** [reply]
test

**Reply by mdestafadilah** [reply]
udah ada komentar da ... :cool:

**Reply by muhammad-rifqi** [reply]
udah si.

**Reply by muhammad-rifqi** [reply]
gw mau nanya nih, maksudnya programmer gadungan apaan , kok bahasanya kayak gitu ??

**Reply by mdestafadilah** [reply]
ya jadi jadian ... asli jadi programmer.. wkwk ...

**Reply by muhammad-rifqi** [reply]
waduuuh wkwkwkwk, kayak penjahaat aja gitu kesannya

**Reply by mdestafadilah** [reply]
kagak lah
